CONTRIBUTING.md

```diff
@@ -53,18 +53,19 @@ On Windows you can also attach from Visual Studio 2017 (Debug | Attach To Proces
 3. Use the `Launch Extension` launch option.
 
 ### Unit Tests
-1. Run the Unit Tests in VS Code via the `Launch Language Server Tests`.
-2. On Windows Open PLS.sln solution in Visual Studio 2017 and run its tests.
-3. Run `dotnet test` from Terminal.
-4. Install .NET Core Test Explorer for VS Code, open src folder in VS Code and run tests.
+To run unit tests, do one of the following:
+- Run the Unit Tests in the [VS Code Python Extension](https://github.com/Microsoft/vscode-python) project via the `Launch Language Server Tests`.
+- On Windows: open PLS.sln solution in Visual Studio 2017 and run tests from the Test Explorer.
+- Run `dotnet test` from Terminal.
```

Comment:
One last feedback, I'd change this line to say:

- Run `dotnet test` in the `src` directory.

Since it has to be run from there.

Reply:
I think actually from `Analysis\Engine\Test` if you want clean run

Reply:
You're right, that actually also gets rid of the annoying messages `dotnet test` produces about not finding tests in the other assemblies. That might be better.